src/architecture/utilities/astroConstants.i

```diff
@@ -0,0 +1,33 @@
+/*
+ ISC License
+
+ Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
update copyright year to 2024

Delete AVS lab reference, please add your name, organization, or leave blank?

Reply:
Done